dashboard/test/models/script_level_test.rb

```diff
@@ -161,4 +161,41 @@ def setup
     assert_equal(script_level3, ScriptLevel.cache_find(script_level3.id))
   end
 
+  test 'has another level answers appropriately for professional learning courses' do
+    create_fake_plc_stuff
+
+    assert @script_level1.has_another_level_to_go_to?
+    assert_not @script_level2.has_another_level_to_go_to?
+  end
+
+  test 'redirects appropriately for professional learning courses' do
+    create_fake_plc_stuff
+
+    assert_equal script_preview_assignments_path(@plc_script), @evaluation_script_level.next_level_or_redirect_path_for_user(@user)
+    @unit_assignment.destroy
+    assert_equal script_stage_script_level_path(@plc_script, @stage, @script_level2.position), @evaluation_script_level.next_level_or_redirect_path_for_user(@user)
+
+    assert_equal script_stage_script_level_path(@plc_script, @stage, @evaluation_script_level.position), @script_level1.next_level_or_redirect_path_for_user(@user)
+    assert_equal script_path(@plc_script), @script_level2.next_level_or_redirect_path_for_user(@user)
+  end
+
+  private
+  def create_fake_plc_stuff
```

Comment:
Nit: method name.  `create_fake_plc_data`?

I also wonder if this much setup code is a sign that our factories should be more comprehensive.